packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3142,6 +3390,131 @@ curl -v -X POST \
 }
 ```
 
+### Verify WebAuthn Factor
+
+<ApiOperation method="post" url="/api/v1/users/${userId}/factors/${factorId}/verify" />
+
+Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options, which are used to help select an appropriate authenticator using the WebAuthn API.
+This authenticator then generates an assertion which may be used to verify the user.
+
+#### Start Verification to Get Challenge Nonce
+
+Verification of the WebAuthn factor starts with getting the WebAuthn credential request details (including the challenge nonce) then using the client-side
+JavaScript API to get the signed assertion from the WebAuthn authenticator.
+
+For more information about these credential request options, see the [WebAuthn spec for PublicKeyCredentialRequestOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions)
```

Comment:
add a period

For more information about these credential request options, see the [WebAuthn spec for PublicKeyCredentialRequestOptions](https://www.w3.org/TR/webauthn/#dictionary-makecredentialoptions).